AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml

```diff
@@ -44,20 +44,24 @@
         <nav class="navbar navbar-default">
             <div class="container-fluid">
                 <div class="navbar-header">
+                    @{
+                        var descriptor = (Microsoft.AspNetCore.Mvc.Controllers.ControllerActionDescriptor)ViewContext.ActionDescriptor;
+                        var areaName = descriptor.RouteValues["area"]; // there must be a better way to get the area name!
+                        var controllerName = descriptor.ControllerName;
+                        var isUser = areaName != "Admin";
+                    }
                     <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                     </button>
-                    <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand"><img src="~/images/Web-Logo-small.png" alt="Web Logo" /></a>
+                    <div @(descriptor.ActionName == "Index" && controllerName == "Home" ? " style=display:none;" : "")>
+                        <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand"><img  src="~/images/Web-Logo-small.png" alt="Web Logo" /></a>
+                    </div> 

```

Comment:
I don't think this change is necessary and it's a little complicated. I think it's totally fine to show the allReady brand image in the navbar on the landing page. This also has the added benefit of visual consistency in the navbar regardless of what view you are on.